examples/music-bot/src/bot.ts

```diff
@@ -0,0 +1,173 @@
+import Discord, { Interaction, GuildMember, Snowflake } from 'discord.js';
+import {
+	AudioPlayerStatus,
+	AudioResource,
+	entersState,
+	joinVoiceChannel,
+	VoiceConnectionStatus,
+} from '@discordjs/voice';
+import { createDiscordJSAdapter } from './music/adapter';
+import { Track } from './music/track';
+import { MusicSubscription } from './music/subscription';
+import { token } from '../auth.example.json';
+
+const client = new Discord.Client({ intents: ['GUILD_VOICE_STATES', 'GUILD_MESSAGES', 'GUILDS'] });
+
+client.on('ready', () => console.log('Ready!'));
+
+// This contains the setup code for creating slash commands in a guild. The owner of the bot can send "!deploy" to create them.
+client.on('message', async (message) => {
+	if (!message.guild) return;
+	if (!client.application?.owner) await client.application?.fetch();
+
+	if (message.content.toLowerCase() === '!deploy' && message.author.id === client.application?.owner?.id) {
```

Comment:
This will now work when the application is owned be a team, I don't think that's too important for this example however.